ui/components/src/model/units.ts

```diff
@@ -1,236 +0,0 @@
-// Copyright 2022 The Perses Authors
```

Comment:
The content of this file was refactored out into a combo of type-specific files and some common helpers in the `units` directory. The file was getting really big, and I'm hoping this will make it easier to read and work with the code.